apps/rush-lib/src/cli/test/Cli.test.ts

```diff
@@ -8,7 +8,7 @@ import { Utilities } from '../../utilities/Utilities';
 describe('CLI', () => {
   it('should not fail when there is no rush.json', () => {
     const workingDir: string = '/';
-    const startPath: string = path.resolve(path.join(__dirname, '../../start.js'));
+    const startPath: string = path.resolve(path.join(__dirname, '../../../lib/start.js'));
```

Comment:
This seems incorrect.  Is it somehow related to the problem from PR https://github.com/microsoft/rushstack/pull/2089 ?

Reply:
Tests under Heft now run in the context of thee files in `src`, not `lib`, so we need to crawl up to `lib` here.